src/App.js

```diff
@@ -20,14 +21,35 @@ const StyledAppContainer = styled.div`
   height: 100vh;
 `
 
+const getTrailData = (trailFilename) => 
+  fetch(`data/trails/${trailFilename}`)
+  .then((res) => res.json())
+  .then((res) => {
+    for (const feature of res.features) {
+      // Search the array and return the trail data
+      if (feature.geometry.type === "MultiLineString") {
+        return feature;
+      }
+    }
+  })
+;
```

Comment:
This works for now, but what if in the future we also wanted to display the point data?

Reply:
we can also use the `.find()` method on Arrays to get the value we want:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
To display point data and the trail data as well, lines 27-34 above would need to be removed since the above only handles returning the trail data. The displayDataOnMap function in the MapArea component would need to loop through the GeoJSON, provided by the getTrailData function specifically line 26, and create/add a source/layer for each feature point data and the trail data.